src/ghdcbot/adapters/github/rest.py

```
@@ -526,6 +528,125 @@ def create_issue_comment(
+            response = self._request(
+                "GET",
+                f"/repos/{owner}/{repo}/contents/{path}",
+                params={},
+            )
+            if response and response.status_code == 200:
+                data = response.json()
+                content_b64 = data.get("content", "")
+                if content_b64:
+                    try:
+                        return base64.b64decode(content_b64).decode("utf-8")
+                    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)